DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Geometry/ISpeckleObjectToHost/CircleToHostConverter.cs

```diff
@@ -19,7 +19,7 @@ public CircleToHostConverter(IRawConversion<SOG.Point, ACG.MapPoint> pointConver
   public ACG.Polyline RawConvert(SOG.Circle target)
   {
     // Determine the number of vertices to create along the cirlce
-    int numVertices = Math.Min((int)target.length, 100); // Determine based on desired segment length or other criteria
+    int numVertices = Math.Max((int)target.length, 100); // Determine based on desired segment length or other criteria
```

Comment:
I bet we should be using `EllipticArcSegment.Densify()` instead of using our own subdivision logic for Arcs/Circles/Ellipses

https://developers.arcgis.com/net/api-reference/api/netwin/Esri.ArcGISRuntime/Esri.ArcGISRuntime.Geometry.Segment.html